### PR TITLE
Add MPR installation instructions

### LINKED ISSUE #22: Add MPR installation instructions
I've added parse-changelog to the [MPR](https://mpr.makedeb.org/packages/parse-changelog), a user-led platform for people to submit and publish packages for Debian and Ubuntu based systems. It's a platform I've personally made, but it's already seen a fair bit of adoption. With that, I was wondering if you'd like to add instructions for installing parse-changelog from the MPR.

If you do decide to go the route of adding the instructions, you may also want to add setup instructions for the [Prebuilt-MPR](https://docs.makedeb.org/prebuilt-mpr/getting-started/), which provides a repository containing prebuilt packages from the MPR (as the MPR itself only contains files to build packages), thought that's up to you.

Regardless of what you choose to do, I also just wanted to say thanks for making this, I just stumbled on this a bit ago and I'm immediately going to start using it in my projects :).

### PR DESCRIPTION
**Pull request by hwittenborn**
Closes #22

### CODE REVIEW

**Comment by hwittenborn** [comment]
Should instructions for installing makedeb also be included? Users might be confused if they don't have it on there system and there wondering why something isn't working.

Should Prebuilt-MPR instructions also be added? It would avoid the need for makedeb to be installed in the first place, and would only require adding an APT repository, which a lot of projects already do

**Review by taiki-e — CHANGES_REQUESTED** [review]
Thanks!

> Should instructions for installing makedeb also be included? Users might be confused if they don't have it on there system and there wondering why something isn't working.

I tend not to include makedeb's installation instructions. Our installation instructions are for users of package managers, and new users of package managers can learn more about it through links to the package.

Also, the package manager's recommended installation instructions may change. (In fact, rustup's recommended installation instructions have been updated in the past for better security. Some projects explain rustup's the old installation instructions in their readme, but that is not a nice situation for new users.)

> Should Prebuilt-MPR instructions also be added? It would avoid the need for makedeb to be installed in the first place, and would only require adding an APT repository, which a lot of projects already do.

Sounds good to have it.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ You can install parse-changelog using [Homebrew tap on macOS and Linux](https://
 brew install taiki-e/tap/parse-changelog
 ```
 
+#### Via MPR (Debian/Ubuntu)
+You can install [parse-changelog from the MPR](https://mpr.makedeb.org/parse-changelog):
```
[the remark this answers is not in the document]
Comment (hwittenborn) [reply README.md:42]: Sorry about that :P I'll get it fixed right now

**Review by taiki-e — APPROVED** [review]
bors r+

**Pull request merged**